components/cache/cache_invalidation.rst

```diff
@@ -0,0 +1,121 @@
+
+    // invalidate all items related to `tag_1`
+    $cache->invalidateTags('tag_2');
+
+    // or invalidate all items related to `tag_1` or `tag_3`
+    $cache->invalidateTags(array('tag_1', 'tag_3'));
+
+Of course when you know the cache key, it's better to call
+``$cache->deleteItem('cache_key');``, but this key is sometimes hard to track
+and this is where cache tags become useful.
+
+Tag Aware Adapters
+~~~~~~~~~~~~~~~~~~
+
+To store tags, you need to wrap a cache adapter with the
+:class:`Symfony\\Component\\Cache\\Adapter\\TagAwareAdapter` class or implement
+:class:`Symfony\\Component\\Cache\\Adapter\\TagAwareAdapterInterface` and its only
+:method:`Symfony\\Component\\Cache\\Adapter\\TagAwareAdapterInterface::invalidateTags`
+method by yourself.
+
+The :class:`Symfony\\Component\\Cache\\Adapter\\TagAwareAdapter` class implements
+instantaneous invalidation. It needs one or two cache adapters: the required one
+is used to store cached items; the second one is used to store tags and their
+invalidation version number (conceptually similar to their latest invalidation
+date). When only one adapter is used, items and tags are all stored in the same
+place. By using two adapters, you can e.g. store some big cached items on the
+filesystem or in the database and keep tags in a Redis database to sync all your
+fronts and have very fast invalidation checks::
+
+    use Symfony\Component\Cache\Adapter\TagAwareAdapter;
+    use Symfony\Component\Cache\Adapter\FilesystemAdapter;
+    use Symfony\Component\Cache\Adapter\RedisAdapter;
+
+    $cache = new TagAwareAdapter(
+        // Adapter for cached items
+        new FilesystemAdapter(),
+        // Adapter for tags
+        new RedisAdapter('redis://localhost')
+    );
+
+Using Cache Namespaces
+----------------------
+
+By using adapters that implement the
+:method:`Symfony\\Component\\Cache\\Adapter\\ForkableAdapterInterface::fork`
+method from :class:`Symfony\\Component\\Cache\\Adapter\\ForkableAdapterInterface`,
+you can create context-dependent variations of your cached items::
+
+    $childCache = $parentCache->fork('child-namespace-name');
+    $siblingCache = $parentCache->fork('sibling-namespace-name');
+    $subChildCache = $childCache->fork('sub-child-namespace-name');
+
+Forks work by cloning existing adapters into new adapters that share everything
+with their parent (esp. the data store connection) but add a ``$namespace``
+prefix to all their keys, in a way that makes a cache key in a forked adapter
+unable to collide with the same key in its parent adapter.
+
+You can use forks everywhere you would otherwise manually prefix your cache keys
+in your code. A typical example is storing different HTML pages per
+``Vary: User-Agent`` HTTP headers: just use one fork per ``Vary`` header.
+
+An added benefit to forks is that clearing them takes nesting into account;
+clearing a parent adapter recursively clears all its forks::
+
+    $childCache->clear();
```

Comment:
how I can clear a fork cache without "dispatch" the clear to all children ? Is it possible ?